modnet/model_presets/presets.py

```diff
@@ -0,0 +1,21 @@
+batch_sizes = [64]
+
+learning_rates = [0.02, 0.005]
+
+archs = [(350,[[64],[8],[8],[]]),(350,[[256],[128],[8],[8]]),(1000,[[256],[128],[8],[]])]
+
+epochs = [1000]
+
+losses = ['mse']
+
+activations = ['elu']
```

Comment:
If `'elu'` is the only activation to test here, should it be the default in the model init (instead of `'relu'`)?